userdocs/src/usage/eks-managed-nodes.md

```diff
@@ -183,6 +183,23 @@ To upgrade to a specific AMI release version instead of the latest version, pass
 eksctl upgrade nodegroup --name=managed-ng-1 --cluster=managed-cluster --release-version=1.19.6-20210310
 ```
 
+!!!note 
+    If the managed nodes are deployed using custom AMIs, the following workflow must me followed in order to deploy to a new version of the custom AMI.
+    - initial deployment of the nodegroup must be done using a launch template. e.g.
+      ```yaml
+      managedNodeGroups:
+        - name: launch-template-ng
+        launchTemplate:
+        id: lt-1234
+        version: "2" #optional (uses the default version of the launch template if unspecified)
```

Comment:
This isn't indented properly.

Reply:
8b1e6eb59bf015fabb4c308fce4a8a169f4e51ca